src/Smart.Design.Library.Showcase.WebApp/Pages/Shared/_ComponentList.cshtml

```diff
@@ -28,6 +28,9 @@
             <div class="br-docs-category-list-wrapper">
                 <ul class="c-styleguide-list">
                     <li class="c-styleguide-list__item @IsActive("Toolbar")"><a asp-page="../../03_cat_ds/Toolbar/Toolbar">Toolbar </a></li>
+                    <li class="c-styleguide-list__item @IsActive("Navbar")"><a asp-page="../../03_cat_ds/Navbar/Navbar">Navbar </a></li>
+                    <li class="c-styleguide-list__item @IsActive("Panel")"><a asp-page="../../03_cat_ds/Pagination/Pagination">Pagination </a></li>
```

Comment:
@IsActive("Panel") et @IsActive("Pagination") sont inversés

Les liens Panel et Pagination  devraient ne plus être affichés dans la section "To order by DS/DL and to document"

Reply:
La première partie du commentaire a été fixé. Pour ce qui est de la deuxième, il y a déjà des liens Panel et Pagination dans la section "To order by DS/DL and to document". Toutefois le contenu de ces pages correspond à la doc .net/DL. Ceux que j'ai mis dans la section "Design System (HTML)" correspond au contenu en HTML. Je pense donc que ces liens doivent rester à cet emplacements.